#include <iostream>
#include <exception>
using namespace std;
class Exception: public exception{

};
int main(void){

    return 0;
}